docs/command_line_reference.rst

```diff
@@ -975,3 +975,19 @@ Examples, assuming that two clusters have been specified with ``--target-hosts``
 
 .. WARNING::
    If you use ``client-options`` you must specify options for **every** cluster name defined with ``target-hosts``. Rally will raise an error if there is a mismatch.
+
+Command line parameters accepting comma-separated values (csv)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Parameters that can accept comma-separated values such as ``--car``, ``--telemetry``, ``--include-tasks`` etc. can also accept a JSON array.
+This can either be defined in a file ending in ``.json`` or passed as an inline JSON string.
+
+Examples:
```

Comment:
Should we also give a comma-separated example?

Reply:
We already do for each individual option, e.g. for [car](https://github.com/elastic/rally/blame/c1bba05b7bbed682063bd7cde8f34af32c2d2cd2/docs/car.rst#L35), but I guess you feel that here it'd make sense to present all possibilities as a convenience (even if it adds some docs duplication)?

Reply:
Right, that's my point, aiming for completeness seems useful. We could even link to this section from the individual options, but that may be too much effort.

I'll trust your judgment on this one, you can leave things as is too.

Reply:
Fixed in 837bd15ba7890f1ba055324855f08d0014e20e98